yaml/parse.ts

```diff
@@ -23,8 +22,10 @@ export interface ParseOptions {
    * Specifies a schema to use.
    *
    * Schema class or its name.
+   *
+   * Passing Schema class is deprecated. Use schema name instead.
```

Comment:
I thought we were no longer doing deprecations. Are we making an exception here? #4908

Reply:
Ok. Let's skip that